### PR TITLE
feat: change specific api

### PR DESCRIPTION
**Pull request by rabbyOnTheMars**

### CODE REVIEW

File: src/ui/views/Dashboard/index.tsx [hunk 1]
```diff
@@ -683,22 +685,32 @@ const Dashboard = () => {
     setTopAnimate('fadeInTop');
   };
   const removeToken = async (removeToken) => {
-    const newAddTokenList = addedToken.filter(
-      (item) => item !== removeToken?.id
+    const uuid = `${removeToken?.chain}:${removeToken?.id}`;
```
Comment (vvvvvv1vvvvvv) [inline src/ui/views/Dashboard/index.tsx:688]: variable `removeToken` use the same name with function `removeToken`, how about changing the variable to `token`?

File: src/ui/views/Dashboard/index.tsx [hunk 2]
```diff
@@ -683,22 +685,32 @@ const Dashboard = () => {
     setTopAnimate('fadeInTop');
   };
   const removeToken = async (removeToken) => {
```
Comment (vvvvvv1vvvvvv) [inline src/ui/views/Dashboard/index.tsx:687]: Please declare type of function arguments if possible

File: src/ui/views/Dashboard/index.tsx (hunk 1 above)
Comment (rabbyOnTheMars) [reply src/ui/views/Dashboard/index.tsx]: done

File: src/ui/views/Dashboard/index.tsx (hunk 2 above)
Comment (rabbyOnTheMars) [reply src/ui/views/Dashboard/index.tsx]: done

**Review by vvvvvv1vvvvvv — APPROVED** [review]

**Pull request merged**